manager/apis/app/v1alpha1/fybrikmodule_types.go

```diff
@@ -40,7 +40,7 @@ const (
 )
 
 // ModuleCapability indicates at a high level what is being performed - often on a datset, but potentially other things as well
-// +kubebuilder:validation:Enum=copy;read;write
+// +kubebuilder:validation:Enum=copy;read;write;transform
```

Comment:
Shouldn't the values be defined in the taxonomy rather than hard-coded enums?

Reply:
Yes, taxonomy issues will be addressed in a separate PR